readme.md

```diff
@@ -16,6 +16,23 @@ console.log(safeStringify(o))  // '{"a":1,"o":"[Circular]"}'
 console.log(JSON.stringify(o)) //<-- throws
 ```
 
+### toJSON support
+
+`fast-safe-stringify` would not attemp to detect circular dependencies
```

Comment:
what about:

 The `toJSON` function is responsible for returning an object to be serialized which represents
the object it's attached to. The general understanding is that the `toJSON` function won't return an
object with circular dependencies. For performance reasons, `fast-safe-stringify` will not attempt to detect circular dependencies on objects with a `toJSON` function. However  this behavior can be overridden on 
a per object basis, by adding a `forceDecirc`property to the `toJSON` function, like so: